extensions/binding/org.eclipse.smarthome.binding.astro/OSGI-INF/.gitignore

```diff
@@ -0,0 +1 @@
+*.xml
```

Comment:
It is quite unexpected from the PR title that you changed to automatic XML generation. Wouldn't it be better to move that to a seperate PR and leave this one for the i18n only?

Reply:
Because I had to reference new OSGi services. It was a perfect opportunity to move to DS annotations.